LondonBritishLibrary/orient/BLorient787.xml

```diff
@@ -0,0 +1,288 @@
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>BL Oriental 787</idno>
+                        <altIdentifier><idno>Wright cat. CCCLV</idno></altIdentifier>
+                        <altIdentifier><idno>Wright 355</idno></altIdentifier>
+                        </msIdentifier>
+
+                        <msContents>
+
+
+
+                          <msItem xml:id="ms_i1">
+                          <locus from="4r"/>
+                          <title type="complete" ref="LIT6128PearlOfGreatPrice"/>
+                          <textLang mainLang="gez"/>
+
+
+                          <msItem xml:id="ms_i1.1">
+                          <locus from="4r"/>
+                          <title type="complete" xml:lang="en">Introduction and table of contents</title>
+                          <incipit xml:lang="gez">
+                          በስመ፡ አብ፡ <gap reason="ellipsis" resp="PRS10171Wright"/> ንወጥን፡ በረድኤተ፡ እግዚአብሔር፡ ልዑል፡ ወሠናይ፡ ሥንዐሁ፡ በጸሒፈ፡
+                          መጽሐፈ፡ ባሕርይ፡ ዘብዙኅ፡ ሤጡ። ዘተጋብአ፡ እምባሕለ፡ ነቢያት፡ ጻድቃን፡ ወሐዋርያት፡ ንጹሓን፡ ወአበው፡ ቅዱሳን፡ ወሊቃነ፡ ጳጳሳት፡ ኅሩያን፡ መምህራነ፡
+                          ቤተ፡ ክርስቲያን፡ እለ፡ ጽኑዐን፡ በመንፈስ፡ ቅዱስ። ይጸግወነ፡ እግዚአብሔር፡ በረከተ፡ ጸሎቶሙ፡ አሜን። ዝንቱ፡ መጽሐፍ፡ ይነግር፡ ቅድመ፡ ተዋሕዶተ፡ ሥላሴሁ፡
+                          ለወልደ፡ እግዚአብሔር፡ ቃል። ወያተሉ፡ ሥጋዌሁ፡ እምእግዝእትነ፡ ማርያም፡ ድንግል፡ እምአሜ፡ ልደቱ፡ እስከ፡ ዕርገቱ፡ ውስተ፡ ሰማይ። ወፈንዎቱ፡ ጰራቅሊጦስሃ፡ ላዕለ፡
+                          ሐዋርያት፡ ንጹሓን፡ ወቅዱሳን። ወይከሥት፡ በውስቴቱ፡ ዝክረ፡ አበው፡ ለለ፡ ፩፡ እምኔሆሙ፡ በበስሙ። ወእምአይ፡ መካን፡ እምጽኡ፡ ስምዐቲሆሙ፡ እንዘ፡ ይከፍል፡ ኀበ፡
+                          ጾታ፡ ፲ወ፪፡ አናቅጸ። ወስብሐት፡ ለእግዚአብሔር፡ እስከ፡ ለዓለም።<lb/>
+                          አንቀጽ፡ ፩፡ በእንተ፡ ሥሉስ፡ ቅዱስ።<lb/>
+                          አንቀጽ፡ ፪፡ በእንተ፡ ልደት፡ ድንግልናዊ።<lb/>
+                          አንቀጽ፡ ፫፡ በእንተ፡ ጥያቄ፡ ተሠግውቱ፡ ወተዋሕዶቱ፡ ኅቡረ።<lb/>
+                          አንቀጽ፡ ፬፡ በእንተ፡ ጥምቀቱ፡ እምዮሐንስ።<lb/>
+                          አንቀጽ፡ ፭፡ በእንተ፡ ዘአመከሮ፡ ሰይጣን፡ በገዳም።<lb/>
+                          አንቀጽ፡ ፮፡ በእንተ፡ ምግባራቲሁ፡ ለእግዚእነ፡ ዘውስተ፡ ወንጌል፡ ቅዱስ፡ እምጊዜ፡ ጥምቀቱ፡ እምዮሐንስ፡ እስከ፡ ጊዜ፡ ሕማማቲሁ፡ ማኅየዊት።<lb/>
+                          አንቀጽ፡ ፯፡ በእንተ፡ ሕማማቲሁ፡ ማኅየዊት፡ በሥጋ።<lb/>
+                          አንቀጽ፡ ፰፡ በእንተ፡ መጥዎተ፡ ነፍሱ፡ መልዕልተ፡ መስቀል።<lb/>
+                          አንቀጽ፡ ፱፡ በእንተ፡ ማይ፡ ወደም፡ ዘውሕዘ፡ እምገቦሁ፡ አምላካዊ።<lb/>
+                          አንቀጽ፡ ፲፡ በንተ፡ ርደቱ፡ ለእግዚእነ፡ በነፍሱ፡ ውስተ፡ ሲኦል፡ እንዘ፡ ይትወሐድ፡ በመለኮት።<lb/>
+                          አንቀጽ፡ ፲ወ፩፡ በእንተ፡ ትንሣኤሁ፡ ቅድስት።<lb/>
+                          አንቀጽ፡ ፲ወ፪፡ በእንተ፡ ዕርገቱ፡ ውስተ፡ ሰማያት።<lb/>
+                          አንቀጽ፡ ፲ወ፫፡ በእንተ፡ ምጽአቱ፡ ዳግም፡ ገሀደ፡ ከመ፡ ይኰንን፡ ሕያዋነ፡ ወሙታነ።<lb/>
+                          አንቀጽ፡ ፲ወ፬፡ በእንተ፡ ዘፈነወ፡ ለጵራቅሊጦስ፡ መንፈሰ፡ ጽድቅ።<lb/>
+                          አንቀጽ፡ ፲ወ፭፡ ቃላት፡ ይነግር፡ በእንተ፡ ሥላሴሁ፡ ቅድስት።<lb/>
+                          ወናሁ፡ ሠራዕነ፡ ወአስተናበርነ፡ ውስተ፡ መጽሐፍነ፡ ዛቲ፡ ወአስተጋባእነ፡ ቃላተ፡ ርቱዐተ፡ ወጽድቃተ፡ በእንተ፡ ሃይማኖት፡ ርትዕት። ወስብሐት፡ ለእግዚአብሔር፡ ጸጋዌ፡ ልቡና፡
+                          ወቃላት፡ ለዓለመ፡ ዓለም፡ አሜን።
+                          </incipit>
```

Comment:
I think it would make sence to move this text to the edition. Because this is actually not an incipit but a citation of the whole textpart, if I understand correctly.
Here you can find further information: https://betamasaheft.eu/Guidelines/?id=text-encoding